api/service/explorer/storage.go

```diff
@@ -20,9 +19,12 @@ import (
 
 // Constants for storage.
 const (
-	AddressPrefix    = "ad"
-	CheckpointPrefix = "dc"
-	PrefixLen        = 3
+	AddressPrefix            = "ad"
+	CheckpointPrefix         = "dc"
+	TransactionPrefix        = "tx"
```

Comment:
What are StakingTransactionPrefix and TransactionPrefix for? Seems they are not used.